app/views/shared/_flash.html.erb

```diff
@@ -1,5 +1,5 @@
 <% flash.each do |type, msg| %>
-  <div class="alert alert-info">
+  <%= content_tag :dev, class: flash_class(type) do %>

```

Comment:
what is dev?

Reply:
actually dev was more beauty than div :laughing: